README.md

```diff
@@ -90,6 +90,16 @@ By default, Carthage will directly [check out][Carthage/Checkouts] dependencies
 
 When run this way, Carthage will write to your repository’s `.gitmodules` and `.git/config` files, and automatically update the submodules when the dependencies’ versions change.
 
+### Automatically rebuilding dependencies
+
+If you want to work on your dependencies during development, and want them to be automatically rebuilt when you build your parent project, you can add a Run Script build phase that invokes Carthage, like so (replacing `iOS` with `Mac` if appropriate):
```

Comment:
Can `iOS` or `Mac` be inferred from build settings? If so, can #302 accept the relevant strings for `--platform`?